.github/workflows/test-repro.yml

```diff
@@ -37,7 +46,7 @@ on:
         value: ${{ jobs.repro.outputs.experiment-location }}
         description: Location of the experiment on the target environment
 env:
-  TEST_OUTPUT_LOCAL_LOCATION: /opt/test-output
+  ARTIFACT_LOCAL_LOCATION: /opt/artifact
```

Comment:
Just curious why this was changed to `/opt/artifact`.

Reply:
Since the overall artifact now has more than just test output, I thought that maybe a different path name was in order, as well :)